D/D.sublime-syntax

```diff
@@ -40,7 +40,7 @@ variables:
   string_lookahead: '(?=`|[rxq]?"|q{)'
 
   definitely_value_lookahead: '(?=!|~|\+|\-|\*|&|\bcast\b|\bdelete\b|\bnew\b|\bimport\b|\bis\b|\b__traits\b|\bfunction\b|\bdelegate\b|[0-9]|\[|\(|{{string_lookahead}}|\b({{language_constant}})\b|\b({{language_variable}})\b)'
-  definitely_declaration_lookahead: '(?=({{name}})(\s+\*|(\*|\[.*\])?\s+)({{name}})|\b({{type_qualifier}})\b)'
+  definitely_declaration_lookahead: '(?=(?:{{name}}\.)*{{name}}(\s+\*+|(?:\*+|\**\[.*\])?\s+){{name}}|\b{{type_qualifier}}\b)'
```

Comment:
Looks like this fails for `foo**bar`?

Reply:
This behavior hasn't changed. It failed before as well as the lookahead enforces whitespace between type and variable.

![grafik](https://user-images.githubusercontent.com/16542113/135648740-5bb263b8-d730-45d4-97e8-89fc671f4e87.png)